README.md

```diff
@@ -1 +1,115 @@
-# json-config-expander
\ No newline at end of file
+# json-config-expander
+
+Expand **multi optional** configuration to **multiple configurations**.
+
```

Comment:
Before the examples, I think you should add a simple explanation of how to install this lib, and on which versions of python it was tested